app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,6 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>LittleEstyShop7</title>
```

Comment:
We still need the line that says "title". That's the website's name that shows up in your browser tab.